pychromecast/socket_client.py

```diff
@@ -204,11 +213,16 @@ def run(self):
                     self.logger.debug("Received: {}".format(
                         _message_to_string(message, data)))
 
-                handled = self._handlers[message.namespace].receive_message(
-                    message, data)
+                try:
+                    handled = self._handlers[message.namespace].receive_message(
+                        message, data)
 
-                if not handled:
-                    self.logger.warning("Message unhandled: {}".format(
+                    if not handled:
+                        self.logger.warning("Message unhandled: {}".format(
+                            _message_to_string(message, data)))
+                except Exception, e:
+                    self.logger.exception(u"Exception {} caught while sending message to controller {}: {}".format(
+                        unicode(e), type(self._handlers[message.namespace]).__name__,
```

Comment:
Yes, you'right, tested it a bit now. In Python 2.x as long as the source string is unicode it will convert things correctly.

Would it be a good idea to prefix all strings (that are not meant to by bytes) with the u literal?